app/utils/bundle-importer.js

```diff
@@ -99,9 +102,11 @@ YUI.add('bundle-importer', function(Y) {
       var notifications = this.db.notifications;
       // If the file passed in was a json file. This should only ever be used
```

Comment:
This comment is difficult for me to parse.
I wonder if we can find a better way to check whether the dropped file is a change set or a bundle yaml.
In theory, we could just use YAML to unmarshal both, and then check the internal structure (obejct vs array?).